client/src/components/PlaceTile.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+const PlaceTile = (props) => {
+  const { distance, id, image, location, name, price, rating, url } = props
```

Comment:
this is totally fine, but you can combine this with line 3 
`const PlaceTile = ({ distance, id, image, location, name, price, rating, url }) => {`